docs/interaction design/meeting prep.tex

```diff
@@ -0,0 +1,105 @@
+\documentclass[a4paper]{article}
+
+\usepackage[english]{babel}
+\usepackage[utf8]{inputenc}
+%styling of author list
+\usepackage{authblk}
+
+
+\title{Interaction Design Meeting}
+
+\author[1]{Louis Gosschalk}
+\author[2]{Boudewijn van Groos}
+\author[3]{Jens Langerak}
+\author[4]{Chris Langhout}
+\author[5]{Paul van Wijk}
+
+\affil[1]{lgosschalk \\
+4214528} 
+\affil[2]{bvangroos \\
+4229843}
+\affil[3]{jlangerak \\
+4317327}
+\affil[4]{clanghout \\
+4281705}
+\affil[5]{pvanwijk \\
+4285034}
+\affil[ ]{Health Informatics Group C}
+
+\date{\today}
+
+\renewcommand*\contentsname{Summary}
+
+\begin{document}
+\maketitle
+
+\section{Relevant}
+the Final Report will also
+contain a section describing the HCI module that was realized for the user interaction with
+the developed solution. This section will reveal what the students learned in the Interaction
+Design course and will be evaluated by the corresponding lecturer. The grade for Interaction
+Design will be assigned based on the content of this section (see Section 8 for assessment
+process and criteria). $\to$ Dit mag dus ongeveer 2 a4'tjes omvatten; zie: \ref{sec:Referentie}:\ref{itm:ite}
```

Comment:
Is inderdaad niet erg duidelijk maar het engels is refererend naar de final report beschrijving van de guidelines